interface/drugs/drug_inventory.php

```diff
@@ -382,50 +382,24 @@ function facchanged() {
 </table>
 
 <!-- TODO: Why are we not using the BS4 table class here? !-->
-<table id='mymaintable' class="display table-striped">
- <thead>
- <tr class='head'>
-  <th>
-    <?php echo xlt('Name'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('Act'); ?>
-  </th>
-  <th>
-    <?php echo xlt('Cons'); ?>
-  </th>
-  <th>
-   <?php echo xlt('NDC'); ?> </a>
-  </th>
-  <th>
-   <?php echo xlt('Form'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('Size'); ?>
-  </th>
-  <th title='<?php echo xlt('Measurement Units'); ?>'>
-    <?php echo xlt('Unit'); ?>
-  </th>
-  <th title='<?php echo xla('Purchase or Transfer'); ?>'>
-    <?php echo xlt('Tran'); ?>
-  </th>
-  <th>
-    <?php echo xlt('Lot'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('Facility'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('Warehouse'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('QOH'); ?> </a>
-  </th>
-  <th>
-    <?php echo xlt('Expires'); ?> </a>
-  </th>
- </tr>
- </thead>
+<table id='mymaintable' class="table table-striped">
+    <thead>
+        <tr>
+            <th><?php echo xlt('Name'); ?> </a></th>
+            <th><?php echo xlt('Act'); ?></th>
+            <th><?php echo xlt('Cons'); ?></th>
+            <th><?php echo xlt('NDC'); ?> </a></th>
+            <th><?php echo xlt('Form'); ?> </a></th>
+            <th><?php echo xlt('Size'); ?></th>
+            <th title='<?php echo xlt('Measurement Units'); ?>'><?php echo xlt('Unit'); ?></th>
```

Comment:
also wrong in original code, but should be using xla for the above title= string